src/main/webapp/js/user-page-loader.js

```diff
@@ -106,10 +106,10 @@ fetch('/blobstore-upload-url')
 /** Fetches data and populates the UI of the page. */
 function buildUI() {
   setPageTitle();
-  showMessageFormIfViewingSelf();
-  fetchMessages();
+  //showMessageFormIfViewingSelf();
+  //fetchMessages();
   fetchAboutMe();
-  fetchBlobstoreUrlAndShowForm()
+  //fetchBlobstoreUrlAndShowForm()
```

Comment:
These lines should be un-commented;  user page cannot display the message and about me forms